scripts/email/execute_template.py

```diff
@@ -24,9 +24,26 @@ def send_emails(bcc_list: List[str]):
     bcc_chunks = chunk_list(bcc_list, 49)
     total_chunks = len(bcc_chunks)
 
-    print(
-        f"Starting email job with {len(bcc_list)} recipients in {total_chunks} batch(es)."
-    )
+    # Add option to list all emails
```

Comment:
Just to clarify, these are print statements for cp devs to make sure the email job is running properly / commands for CP devs to follow when running the email job, correct? If so, do we get warnings if the email is not sent to certain users / can we add those checks?

Reply:
Argh I don't get notifications for replies 💀 sorry for always being delayed in my responses here.

Yes, this is just general notifications. Added in a warning output.